SpringProject/src/main/resources/db/migration/V3__create_schema.sql

```diff
@@ -0,0 +1,5 @@
+CREATE TABLE tokens (
```

Comment:
What was the reason behind extracting tokens in a separate table? Do you really need to normalise this table? Remember that each join is a time consuming operation.

Reply:
I think it's so we can have a separate uuid for each session which are posted and deleted on login and logout respectively

Reply:
Harry has explained that the reason we are doing it in the way we are is so that users can be logged in on multiple devices at once.